src/main/java/org/reaktivity/nukleus/kafka/internal/stream/KafkaCacheClientFetchFactory.java

```diff
@@ -235,7 +236,8 @@ public MessageConsumer newStream(
                             authorization,
                             affinity,
                             partition,
-                            defaultOffset);
+                            defaultOffset,
+                            DEFAULT_LATEST_OFFSET);
```

Comment:
If we always pass `DEFAULT_LATEST_OFFSET` then, let's remove the constructor parameter and assign to the default value directly in the constructor instead.